odoo/addons/base/models/res_users.py

```diff
@@ -679,7 +679,7 @@ def check_super(self, passwd):
     @api.model
     def _get_invalidation_fields(self):
         return {
-            'groups_id', 'active', 'lang', 'tz', 'company_id',
+            'groups_id', 'active', 'lang', 'tz', 'company_id', 'company_ids',
```

Comment:
Not in the right commit. (Should be in https://github.com/odoo/odoo/pull/88276/commits/98d596de7f2ef08e518a1b366a50f1f7a34a2e12)